ansible_collections/arista/avd/plugins/plugin_utils/utils/python_to_ansible_logging_handler.py

```diff
@@ -25,20 +25,27 @@ class PythonToAnsibleHandler(Handler):
     * send DEBUG logs to display.vvv which can be visualized when running a playbook with `-vvv`
     """
 
-    def __init__(self, result: dict, display: Display) -> None:
+    def __init__(self, result: dict | None, display: Display | None = None) -> None:
```

Comment:
Actually because it is a singleton we should just remove it from input and always get it from there

Reply:
will be addressed in the logging PR